src/_sass/_base.scss

```diff
@@ -50,29 +50,7 @@ a {
 img {
   display: block;
   max-width: 100%;
+  height: auto;
   margin: 0 0 1rem;
   border-radius: 5px;
 }
-
-table {
-  margin-bottom: 1rem;
-  width: 100%;
-  font-size: 85%;
-  border: 1px solid #e5e5e5;
-  border-collapse: collapse;
-}
-
-td,
-th {
-  padding: .25rem .5rem;
-  border: 1px solid #e5e5e5;
-}
-
-th {
-  text-align: left;
-}
-
-tbody tr:nth-child(odd) td,
-tbody tr:nth-child(odd) th {
-  background-color: #f9f9f9;
-}
```

Comment:
Doesn't seem to be used 👍

Reply:
Moved it to tabless.scss, for responsiveness. But seems not possible in .md files without adding custom HTML. I prefer tables in a separate file, though.

Reply:
I think that moving this table styling to its own file (tables.css) should be possible anyhow 👍🏽 

Unsure if custom HTML is needed (within .md files). Because I haven't looked into the HTML code that is generated from the "Markdown table syntax".